index.js

```diff
@@ -141,3 +142,12 @@ module.exports = (archive, target, opts, cb) => {
   return status
 }
 
+function joinHyperPath (base, path) {
+  path = join(base, path)
+  if (path === '.') {
+    // '.' is returned when base is '' and path is '': aka, the root directory
+    // in hyperdrive, root should be '' or '/', so we replace it with this special case
+    return ''
+  }
+  return path
```

Comment:
That should work, though I think it'd be a change from how the importer has been doing things. The paths so far have been, eg:

''
'foo.txt'
'foo/bar.txt'

That default would cause:

'/'
'foo.txt'
'foo/bar.txt'